src/spots/spots-views.controller.ts

```diff
@@ -221,6 +221,12 @@ export class SpotsViewsController {
 			return res.redirect(`/spots/${id}/configure`);
 		}
 
+		if (
+			spotWithUser.directAccessToWebsiteTarget &&
+			spotWithUser.websiteTarget !== null
```

Comment:
I thought about using `!=` instead of `!==` to include `undefined`. But there is no reason for this value to be `undefined`.

It is the first time we verify if something is `null` in a controller.